graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,30 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+
+export const modules = async () => {
+  return prisma.module.findMany({
+    include: {
+      author: true,
+      lesson: true
+    }
+  })
+}
+
+export const addModule = async (_parent: void, args: MutationAddModuleArgs) => {
+  const { authorId, content, lessonId, name } = args
```

Comment:
we shouldn't have `authorId` be passed in, it should be retrieved from the cookie.

Reply:
Is authorId the same as userId?
If we can assume the user will always be logged in when calling this resolver, and if `authorId=userId`, then we actually have it in `req.session` already:
https://github.com/garageScript/c0d3-app/blob/4b140cb4926a4c92ffcbc782a972e44b64185fc2/helpers/controllers/passwordController.ts#L101

If we can't assume the user will be logged on while making a request so this resolver, then

Reply:
If authorId=userId, then we set in in `req.session` already
https://github.com/garageScript/c0d3-app/blob/4b140cb4926a4c92ffcbc782a972e44b64185fc2/helpers/controllers/authController.ts#L54

Reply:
Hmm I don't think authorId=userId, so that probably won't work.